build/charts/flow-aggregator/values.yaml

```diff
@@ -45,8 +45,25 @@ flowCollector:
 clickHouse:
   # -- Determine whether to enable exporting flow records to ClickHouse.
   enable: false
-  # -- DatabaseURL is the url to the database. TCP protocol is required.
+  # -- DatabaseURL is the url to the database. Provide the database URL as a string with format
+  # <Protocol>://<ClickHouse server FQDN or IP>:<ClickHouse port>. The protocol has to be one of
+  # the following: "tcp", "tls", "http", "https". When "tls" or "https" is used, tls will be enabled.
   databaseURL: "tcp://clickhouse-clickhouse.flow-visibility.svc:9000"
+  # -- tls contains tls related configuration options, which is used to connect to clickhouse service.
+  tls:
+    # -- Determine whether to skip the verification of the server's certificate chain and host name. Default is false.
+    insecureSkipVerify: false
+    # -- Indicates whether to use custom CA certificate. Default root CAs will be used if this field is false.
+    # If true, a Secret named "clickhouse-ca" must be provided with the following keys:
+    # ca.crt: <CA certificate>
+    caCert: false
+  # -- HostAliases contains a mapping (Hostname to IP), which is used when connecting flow-aggregator
+  # to an ingress. This mapping will be added to /etc/hosts in flow-aggregator.
+  # This field is mainly for e2e test. User can also add hostAliases through the flow-aggregator
+  # deployment
+  hostAliases:
+    ip: ""
+    hostname: ""
```

Comment:
we should avoid adding test-specific values to this chart

I can think of several options:
1) leverage `ServerName` is the TLS config: https://pkg.go.dev/crypto/tls#Config. This lets you access the service using a different name. This configuration could be exposed as part of the TLS options here.
2) access the service using the IP address directly, and include the IP address in the certificate (this is tricky if you cannot get the IP address ahead of time).
3) use the standard `hostAliases` configuration field in the Pod spec, and expose that directly in this chart. https://kubernetes.io/docs/tasks/network/customize-hosts-file-for-pods/

Reply:
I tried the `ServerName `, but for some reason, it still can't work with ingress. I think we will need service ip beforehand for the 2ed and 3rd option? And it might be difficult if we want to get the service ip and use it in the certificate or yaml file?

As a result, I use `AppsV1().Deployments()` in the e2e test to get the Flow Aggregator Deployment and update the `hostAliases ` there. Do we have any concern when updating pod spec by this way?